app/models/food_truck.rb

```diff
@@ -14,4 +14,18 @@ class FoodTruck < ActiveRecord::Base
   def self.search(query)
     where("name ilike ?", "%#{query}%")
   end
+
+  def self.avg_rating(food_truck)
+    food_truck_reviews = food_truck.reviews
+    if food_truck_reviews.length == 0
+      return 0
+    else
+      sum = 0
```

Comment:
You could do something like `food_truck_reviews.inject(:+)` and get rid of 23-26.